<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_a_point_road_sign_Form</class>
 <widget class="QWidget" name="Add_a_point_road_sign_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Add_a_point_road_sign</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/Add_a_point_road_sign_ICON.png</normaloff>../img/Add_a_point_road_sign_ICON.png</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Start_date_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>600</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">start date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Width_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>600</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string notr="true">width:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Width_lineEdit">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>600</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="Comments_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>640</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">comments:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Comments_textEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>640</y>
     <width>630</width>
     <height>55</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Height_label">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>600</y>
     <width>38</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">height:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Height_lineEdit">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>600</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="Angle_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>600</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">angle:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Angle_lineEdit">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>600</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="About_pushButton">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>5</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">about</string>
   </property>
  </widget>
  <widget class="QLabel" name="Type_groups_folder_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">type groups folder</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string notr="true"> ---------------------------------------------------------------------------------------------------------------------------------------------------------------</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Type_preview_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>5</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">type preview</string>
   </property>
   <property name="text">
    <string notr="true">preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Set_type_groups_folder_pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>5</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">set type groups folder</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Start_dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>600</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2019</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="Type_group_preview_scrollArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>89</y>
     <width>690</width>
     <height>500</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>498</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>671</width>
       <height>481</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Selected_type_file_name_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>65</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">type = selected type file name</string>
   </property>
   <property name="text">
    <string notr="true"> -------------------------------------------------------------------------------------------------------------------------------------------------------------------</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Type_groups_comboBox">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>5</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">type groups</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Type_filter_pushButton">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>60</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Manual_pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>5</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">manual</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Memory_3_pushButton">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>599</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">memory no 3</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="Memory_2_pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>599</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">memory no 2</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="Memory_1_pushButton">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>599</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">memory no 1</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="Set_default_pushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>599</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">set default from Setup file</string>
   </property>
   <property name="text">
    <string notr="true">Df</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Del_memory_1_pushButton">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>623</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">delete memory no 1</string>
   </property>
   <property name="text">
    <string notr="true">del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Del_memory_2_pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>623</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">delete memory no 2</string>
   </property>
   <property name="text">
    <string notr="true">del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Del_memory_3_pushButton">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>623</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">delete memory no 3</string>
   </property>
   <property name="text">
    <string notr="true">del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Set_angle_pushButton">
   <property name="geometry">
    <rect>
     <x>491</x>
     <y>599</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">set angle (push LMB, move cursor, release LMB)</string>
   </property>
   <property name="text">
    <string notr="true">sa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Copy_data_pushButton">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>599</y>
     <width>30</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">select an object first and then push this button to copy data</string>
   </property>
   <property name="text">
    <string notr="true">cp</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Graphics_type_groupBox">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>1</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true">raster is enabled in QGIS version &gt;= 3.6</string>
   </property>
   <property name="title">
    <string notr="true">graphics</string>
   </property>
   <widget class="QRadioButton" name="SVG_radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">svg</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="Raster_radioButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">raster</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="Street_name_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>700</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">street:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Street_name_textEdit">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>700</y>
     <width>630</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
